Dockerfile

```diff
@@ -36,4 +38,5 @@ RUN groupadd -r mapproxy -g 10001 && \
 RUN chown -R mapproxy:mapproxy /mapproxy /settings /start.sh
 VOLUME [ "/mapproxy"]
 USER mapproxy
-CMD /start.sh
+ENTRYPOINT [ "/start.sh" ]
+CMD ["mapproxy-util", "serve-develop", "-b", "0.0.0.0:8080", "mapproxy.yaml"]
```

Comment:
Set `/start.sh` as the default entrypoint. This way we can supply alternative command args for debugging purpose (in case we need to run different args, like `tail -f /dev/null` to let the container run indefinitely).

Set `CMD` to the command being run if not in prod mode. It's a bit redundant but we can choose either to let the prod mode become the default `CMD` or the development one. Here I choose the development mode as the default `CMD` arg because we are using `PRODUCTION=True` environment flag and being handled directly in `/start.sh`.

To describe what happen with the alternative:
If we set `CMD` to `uwsgi --ini /settings/uwsgi.ini`, then in `/start.sh` we need to change the flag to check if `PRODUCTION=False` then run `mapproxy-util` command argument. Otherwise, let the entrypoint run whatever it is the `CMD` defined in `docker-compose.yaml` file.

Reply:
So setting Prodction=false will run the dev mode?

Reply:
> So setting Prodction=false will run the dev mode?
Yes, other than True it will use dev mode if supplied with no args. When supplied with command args, it will run that instead in Production=False. 

In the Dockerfile, we set up PRODUCTION=True, so if we not define PRODUCTION in docker-compose, it will use PRODUCTION=true.